docs/topics/development/remote_settings.md

```diff
@@ -0,0 +1,116 @@
+# Remote Settings
+
+This page explains how to set up [Remote Settings][] locally, which can be
+useful when working on the [Blocklist feature](../blocklist.md). AMO must be
+running locally first. Make sure it's available at <http://olympia.test/>.
+
+## Configure addons-server
+
+Add the following configuration variables to the `local_settings.py` file:
+
+```
+# When using Docker Desktop - `host.docker.internal` is a special host to allow
+# containers to get access to the host system, but that won't work on Linux.
+REMOTE_SETTINGS_API_URL = "http://host.docker.internal:8888/v1/"
+REMOTE_SETTINGS_WRITER_URL = "http://host.docker.internal:8888/v1/"
+
+# For Linux, you need to find the IP address of the Remote Settings container:
+# REMOTE_SETTINGS_API_URL = "http://172.17.0.1:8888/v1/"
+# REMOTE_SETTINGS_WRITER_URL = "http://172.17.0.1:8888/v1/"
+```
+
+Next, reload everything by running `make up`.
+
+At this point, AMO should be able to find the Remote Settings local server that
+we're going to set up next.
+
+## Set up Remote Settings
+
+In order to set up Remote Settings, follow these steps:
+
+1. Clone <https://github.com/mozilla/remote-settings>
+2. Run `make start`
+3. Add `127.0.0.1 autograph` to your `/etc/hosts` file
+
+Verify that Remote Settings is healthy:
```

Comment:
you know... it'd be great if we had a script to do all this so we didn't need to copy and paste all the random setup? 😉

Reply:
ha well, how strongly do you feel about this? I kinda like step-by-step guides

Reply:
I can see the value in explaining step by step how it works.... though personally, as it's not a core part of AMO, I'm more interested in the end result of a working remote settings setup 🤷

Reply:
Understood. I am going to keep it as is for now. I've tried to reduce the nunmber of steps as much as possible.